### PR TITLE
Prettier setup

### PR DESCRIPTION
**Pull request by alliejones**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,27 +47,28 @@ The following commands will all need to be run in a terminal/command line prompt
...
+
+_Note: Please use the two-letter [ISO-639-1 language code](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes). If your content is targeted to a specific country as well, use the format of `pt-BR` using the [ISO-3166 country code](https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes)._
 
 ### Start by creating the translated content
 
-1. Find the `/en/` folder in the directory and make a copy of it (and all of it's children) renamed with the two-letter [ISO-639-1 language code](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) of your language - e.g. `/de/` for German. In this folder you will find copies of the english-language files for the site, broken down into sub-groups of files with `/swears/` and `/noswears/`. 
+1. Find the `/en/` folder in the directory and make a copy of it (and all of it's children) renamed with the two-letter [ISO-639-1 language code](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) of your language - e.g. `/de/` for German. In this folder you will find copies of the english-language files for the site, broken down into sub-groups of files with `/swears/` and `/noswears/`.
```
Comment (alliejones) [inline README.md:61]: Prettier obviously decided to reformat all of the markdown. I set it preserve line breaks but clearly a bunch of stuff changed, would you rather just exclude md from prettier?

Comment (ksylor) [reply README.md:61]: haha yeah that seems best!

**Comment by alliejones** [comment]
Updated to exclude `.md` and `.html`.

**Review by ksylor — APPROVED** [review]
Yay!! Thank you, Allie!

**Pull request merged**